website/docs/docs/build/incremental-microbatch.md

```diff
@@ -179,12 +179,18 @@ It does not matter whether the table already contains data for that day. Given t
 
 Several configurations are relevant to microbatch models, and some are required:
 
-| Config   | Type | Description   | Default |
-|----------|------|---------------|---------|
-| [`event_time`](/reference/resource-configs/event-time) | Column  (required)   | The column indicating "at what time did the row occur." Required for your microbatch model and any direct parents that should be filtered.          | N/A     |
-| `begin`      | Date (required)   | The "beginning of time" for the microbatch model. This is the starting point for any initial or full-refresh builds. For example, a daily-grain microbatch model run on `2024-10-01` with `begin = '2023-10-01` will process 366 batches (it's a leap year!) plus the batch for "today."        | N/A     |
-| `batch_size` | String (required)  | The granularity of your batches. Supported values are `hour`, `day`, `month`, and `year`             | N/A     |
-| `lookback`   | Integer (optional) | Process X batches prior to the latest bookmark to capture late-arriving records.                                         | `1`     |
+| Config   |  Description   | Default | Type | Required  |
+|----------|---------------|---------|------|---------|
+| [`event_time`](/reference/resource-configs/event-time)  | The column indicating "at what time did the row occur." Required for your microbatch model and any direct parents that should be filtered.   | N/A     |  Column  |  Required |
+| `begin`      |  The "beginning of time" for the microbatch model. This is the starting point for any initial or full-refresh builds. For example, a daily-grain microbatch model run on `2024-10-01` with `begin = '2023-10-01` will process 366 batches (it's a leap year!) plus the batch for "today."        | N/A     | Date   | Required |
+| `batch_size` |  The granularity of your batches. Supported values are `hour`, `day`, `month`, and `year`    | N/A     | String  | Required |
+| `lookback`   | Process X batches prior to the latest bookmark to capture late-arriving records.    | `1`     | Integer | Optional |
+| `unique_key` |  A column(s) (string or array) or expression for the record. Required for the `check` strategy. | N/A | String <br />  | Optional* |
+| `partition_by` | A column(s) (string or array) or expression for the record. Required for the `check` strategy. | N/A | String | Optional* |
```

Comment:
Where did the
> Required for the `check` strategy.

part of the description come from? It feels odd here, as in, what is a `check` strategy in context of defining a microbatch model?

Additionally, I feel weird about including `unique_key` and `partition_by` in the table. They are only relevant for specific adapters, and not microbatch as a whole. The issue is the microbatch isn't a uniform strategy across the different adapters. That is (where `~=` means "approximately equals")
* dbt-postgres `microbatch` ~= `merge`
* dbt-redshift `microbatch` ~= `delete+insert`
* dbt-bigquery `microbatch` ~= `insert_overwrite`
* dbt-spark `microbatch` ~= `insert_overwrite`
* dbt-databricks `microbatch` ~= `insert_overwrite`
* dbt-snowflake `microbatch` ~= `delete+insert`

The `unique_key` is required for dbt-postgres's `microbatch` because it's `merge` strategy requires a `unique_key`. And that `unique_key` is used to identify _which_ rows in the data warehouse need to get merged. For dbt-bigquery and dbt-spark, their underlying implementation of `insert_overwrite` requires a `partition_by`. My understanding of that strategy is that `partition_by` makes it _efficient_.

Reply:
Oh thanks for catching this! this def shouldn't be there - it was tricky and maybe we need a separate 'adapters-related' section to explain the microbatch x adapter strategy. 

I'll rejig this !